### PR TITLE
CI pipeline improvements

### PR DESCRIPTION
**Pull request by Skyld**
Purpose of this PR is to slightly improve the performance of our CI pipeline by parallelising the steps within it. Now the pipeline will work like this:

Screenshot 2021-09-10 at 16 13 05

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -51,6 +51,12 @@
     "conflict": {
         "symfony/symfony": "*"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/theofidry/AliceBundle"
+        }
+    ],
```
Comment (luisfn) [inline composer.json:59]: Why this is needed now?

**Review by luisfn — COMMENTED** [review]
Thanks Dani, really liked how CI/CD is organized now.

My only concern is the time it's taking for the Infection test stage, this is related only to environment performance?

Also, why we need the extra repository now but not before?

File: composer.json (hunk 1 above)
Comment (Skyld) [reply composer.json:59]: Oh yes, I forgot to explain it in the PR description. It is required due to this unfortunate "incident": https://github.com/nelmio/alice/issues/1089

**Comment by Skyld** [comment]
> Thanks Dani, really liked how CI/CD is organized now.
> 
> My only concern is the time it's taking for the Infection test stage, this is related only to environment performance?
> 
> Also, why we need the extra repository now but not before?

I also noticed the slowliness, I think it is caused by the limitations of the CI server. I agree that we should investigate it together with @serge-salamanka-1pt

**Review by luisfn — APPROVED** [review]
Looks good to me!

**Review by uncleempty — APPROVED** [review]
:+1: 
LGTM

**Pull request merged**